src/main.cpp

```diff
@@ -5,6 +5,8 @@
 #include "include/parser.hpp"
 #include "include/parsing_visualizer.hpp"
 
+#define PARSING_VISUALIZATION_ENABLED
```

Comment:
OK, but should be fixed in next commits. We won't merge until this is already fixed

Reply:
This flag is set only for demonstration purpuses. In future releases an argument parser would be a really good addition. So would you be OK, if this is remains unchanged for now.

Reply:
OK